.changeset/young-masks-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@compiled/babel-plugin': minor
+---
+
+Implemented babel-plugin 0.19.0 patch
```

Comment:
Nit: perhaps something more descriptive, like "Added error in debug mode when using 'innerRef' instead of 'ref'"

You'll see these descriptions included in our version releases: https://github.com/atlassian-labs/compiled/releases

Reply:
(nit means [nitpick](https://curtiseinsmann.medium.com/how-to-nitpick-on-code-reviews-with-empathy-d6cb8e670ca4))

Reply:
Thanks! I'll add this now :))

Reply:
no prob! your PR looks good :)